src/app/settings/shelf-settings/components/shelf-action-panel-settings-page/shelf-action-panel-settings-page.component.html

```diff
@@ -27,13 +27,15 @@
                 <td>
                     {{ actionPanel.rank }}
                 </td>
-                <td>
-                    <sofie-rounded-box *ngFor="let filter of actionPanel.actionFilter" class="{{ filter | actionColorClass }} me-2">
+                <td >
+                  <div class="flex" style="flex-wrap: wrap; gap: 8px;">
```

Comment:
I personally don't see the point of inline styles since we already have a class defined and we can add specific one for this inline styles. And in general I don't think they are a good practice.

Reply:
The inline styling has been moved to the dedicated class `action-filters`.

Reply:
Just to note for everyone: We want to avoid inline styling whereever possible.